setup_env.sh

```diff
@@ -34,25 +33,37 @@ libgtest-dev
 )
 
 if [ $DISTRIB_RELEASE!="14.04" ]; then
+   #add-apt-repository -y ppa:smspillaz/cmake-2.8.12
```

Comment:
@ovvasyliev please remove commented code.Why you prefer use sed instead of ppa add?